op-service/txmgr/txmgr.go

```diff
@@ -562,38 +616,77 @@ func (m *SimpleTxManager) increaseGasPrice(ctx context.Context, tx *types.Transa
 		m.l.Info("re-estimated gas differs", "tx", tx.Hash(), "oldgas", tx.Gas(), "newgas", gas,
 			"gasFeeCap", bumpedFee, "gasTipCap", bumpedTip)
 	}
-	rawTx.Gas = gas
+
+	var newTx *types.Transaction
+	if tx.Type() == types.BlobTxType {
+		// Blob transactions have an additional blob gas price we must specify, so we must make sure it is
+		// getting bumped appropriately.
+		bumpedBlobFee := calcThresholdValue(tx.BlobGasFeeCap(), true)
+		if bumpedBlobFee.Cmp(blobBasefee) < 0 {
+			bumpedBlobFee = blobBasefee
+		}
+		maxBlobFee := new(big.Int).Mul(calcBlobFeeCap(blobBasefee), big.NewInt(int64(m.cfg.FeeLimitMultiplier)))
+		if bumpedBlobFee.Cmp(maxBlobFee) > 0 {
+			m.l.Warn("bumped blob fee getting capped at multiple of the suggested value", "bumped", bumpedBlobFee, "suggestion", maxBlobFee)
+			bumpedBlobFee.Set(maxBlobFee)
+		}
```

Comment:
would be great to extend the existing fee bumping test to cover this code

Reply:
Hmm, while this code was pending we went from capping without error to returning errors for regular fee capping, I guess I should make hitting the blob fee cap also error out?

Reply:
OK changed it to return error should the blob fee cap get hit instead of silent capping.

Also extended the fee bumping test to confirm blob transactions appropriately error out, but as of now I'm not sure how to make it hit this limit before it hits the regular fee cap limit, so not quite exercising this code path!

Reply:
Fixed the test to confirm blob fee capping works as expected.